gpkit/small_scripts.py

```diff
@@ -37,7 +42,8 @@ def unitstr(units, into="%s", options="~", dimless='-'):
         except ValueError:
             rawstr = "1.0 " + str(units.units)
         units = "".join(rawstr.replace("dimensionless", dimless).split()[1:])
-    return into % units if units else ""
+        return into % units if units else ""
+    return ""
```

Comment:
A suggestion for cleaning this up is on the `unitstrcleanup` branch. Please treat it like a PR and merge it into this branch if you like it.